### PR TITLE
Add support for scoped packages

### LINKED ISSUE #17: Support for scoped packages
I was just searching names for my project when I saw that this package does not support scoped packages.

Check https://runkit.com/embed/mzsivagq3l35 for a demo.

### PR DESCRIPTION
**Pull request by clemtrek**
Fixes #17

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -17,3 +17,12 @@ test('returns a map of multiple package names', async t => {
 	t.false(res.get(name1));
 	t.true(res.get(name2));
 });
+
+test('returns true when scoped package name is not taken', async t => {
+	t.true(await m(`${uniqueString()}/${uniqueString()}`));
```
Comment (sindresorhus) [inline test.js]: You forgot the `@`

Comment (clemtrek) [reply test.js]: Yeah! Sorry. I will fix this

**Comment by clemtrek** [comment]
@sindresorhus, I think all should be fine now.

**Comment by sindresorhus** [comment]
Looks good :)

**Pull request merged**